diff --git a/node_modules/@interchain-kit/store/esm/wallet-manager/chain-wallet-store.js b/node_modules/@interchain-kit/store/esm/wallet-manager/chain-wallet-store.js
index 56dc3d3..2d47c90 100644
--- a/node_modules/@interchain-kit/store/esm/wallet-manager/chain-wallet-store.js
+++ b/node_modules/@interchain-kit/store/esm/wallet-manager/chain-wallet-store.js
@@ -90,13 +90,13 @@ export class ChainWalletStore extends BaseWallet {
         const aminoOfflineSigner = {
             getAccounts: async () => [account],
             signAmino: async (signer, signDoc) => {
-                return cosmosWallet.signAmino(this.chain.chainId, signer, signDoc, {});
+                return cosmosWallet.signAmino(this.chain.chainId, signer, signDoc, undefined);
             }
         };
         const directOfflineSigner = {
             getAccounts: async () => [account],
             signDirect: async (signer, signDoc) => {
-                return cosmosWallet.signDirect(this.chain.chainId, signer, signDoc, {});
+                return cosmosWallet.signDirect(this.chain.chainId, signer, signDoc, undefined);
             }
         };
         const signType = preferSignType || preferredSignTypeFromSettings;
diff --git a/node_modules/@interchain-kit/store/wallet-manager/chain-wallet-store.js b/node_modules/@interchain-kit/store/wallet-manager/chain-wallet-store.js
index c23dded..145d079 100644
--- a/node_modules/@interchain-kit/store/wallet-manager/chain-wallet-store.js
+++ b/node_modules/@interchain-kit/store/wallet-manager/chain-wallet-store.js
@@ -93,13 +93,13 @@ class ChainWalletStore extends core_1.BaseWallet {
         const aminoOfflineSigner = {
             getAccounts: async () => [account],
             signAmino: async (signer, signDoc) => {
-                return cosmosWallet.signAmino(this.chain.chainId, signer, signDoc, {});
+                return cosmosWallet.signAmino(this.chain.chainId, signer, signDoc, undefined);
             }
         };
         const directOfflineSigner = {
             getAccounts: async () => [account],
             signDirect: async (signer, signDoc) => {
-                return cosmosWallet.signDirect(this.chain.chainId, signer, signDoc, {});
+                return cosmosWallet.signDirect(this.chain.chainId, signer, signDoc, undefined);
             }
         };
         const signType = preferSignType || preferredSignTypeFromSettings;
